UICatalog/UICatalog.cs

```diff
@@ -154,11 +155,15 @@ private static Scenario GetScenarioToRun ()
 
 			_menu = new MenuBar (new MenuBarItem [] {
 				new MenuBarItem ("_File", new MenuItem [] {
-					new MenuItem ("_Quit", "", () => Application.RequestStop() )
+					new MenuItem ("_Quit", "", () => Application.RequestStop(), null, null, Key.ControlQ)
```

Comment:
In this case that doesn't happen because in the end it's translates to` Ctrl+'A'`, but the `Key.CtrlMask` must be assigned to inject into the key.

Reply:
Because of this I made the suggestion to change the `Key.Control<char>` to only the` Key.<char>`, e.g. from` ControlA = 1` to `A = 65` and change the verification key in the case of `Ctrl+A` change from:

```
} else if (wch2 - (int)Key.Space >= 'A' && wch2 - (int)Key.Space <= 'Z') {

} else if (wch2 >= (uint)Key.ControlA && wch2 <= (uint)Key.ControlZ) {
```

to:

`} else if (wch2 - (int)Key.Space >= (uint)Key.A && wch2 - (int)Key.Space <= Key.Z) {

} else if (wch2 >= (uint)Key.A - 64 && wch2 <= (uint)Key.Z - 64) {`

What do you think?